### PR TITLE
Added data on package module that allows to avoid duplicate tasks usi…

### PR DESCRIPTION
**Pull request by dorsev**
…ng when's and does all the magic inside for you

### CODE REVIEW

File: workshop/3_Nginx-the-plot-thickens.md [hunk 1]
```diff
@@ -64,6 +64,19 @@ That's the *Install nginx on RedHat/CentOS* task that is not running because the
 
 AHHH.
 
+#### Wait again. Isn't there a better solution than duplicating my tasks?
+ofcourse there is.
```
Comment (erikzaadi) [inline workshop/3_Nginx-the-plot-thickens.md]: Nitbit:
`ofcourse` => `of course`

File: workshop/3_Nginx-the-plot-thickens.md [hunk 2]
```diff
@@ -64,6 +64,19 @@ That's the *Install nginx on RedHat/CentOS* task that is not running because the
...
 
+#### Wait again. Isn't there a better solution than duplicating my tasks?
+ofcourse there is. 
+	
+
+```
+...
+  tasks:
+    - name: Install nginx on Ubuntu/Debian/RedHat/CentOS
+      package:  name=nginx state=present
```
Comment (erikzaadi) [inline workshop/3_Nginx-the-plot-thickens.md]: 2 spaces after `package:`

**Review by erikzaadi — COMMENTED** [review]
Awesomico! 2 small grammarish comments

**Comment by dorsev** [comment]
@erikzaadi , should I squash both commits into one?

**Comment by erikzaadi** [comment]
Squash plz

**Comment by dorsev** [comment]
@erikzaadi done

**Review by erikzaadi — APPROVED** [review]

**Pull request merged**